src/CONFIG.js

```diff
@@ -1,33 +1,31 @@
-import _ from 'underscore';
 import lodashGet from 'lodash.get';
 import {Platform} from 'react-native';
 import Config from 'react-native-config';
 import getPlatform from './libs/getPlatform/index';
 
-// Set default values to contributor friendly values to make develo:pment work out of the box without an .env file
-let expensifyCashURL = lodashGet(Config, 'URL_EXPENSIFY_CASH', 'https://expensify.cash/');
-let expensifyURL = lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/');
-let ngrokURL = lodashGet(Config, 'NGROK_URL', null);
-const useNgrok = lodashGet(Config, 'USE_NGROK', 'false');
-const useWebProxy = lodashGet(Config, 'USE_WEB_PROXY', 'true');
-const expensifyComWithProxy = getPlatform() === 'web' && useWebProxy === 'true' ? '/' : expensifyURL;
-
-// Let's make everyone's life just a bit easier
-// by adding / to the end of any config URL's if it's not already present
-if (_.isString(ngrokURL) && !ngrokURL.endsWith('/')) {
-    ngrokURL += '/';
-}
-if (!expensifyCashURL.endsWith('/')) {
-    expensifyCashURL += '/';
-}
-if (!expensifyURL.endsWith('/')) {
-    expensifyURL += '/';
+/**
+ * Let's make everyone's life just a bit easier by adding / to the end of any config URL's if it's not already present
+ * @param {string} URL
+ * @returns {string}
+ */
+function wrapWithBackslash(URL) {
+    if (!URL.endsWith('/')) {
+        return `${URL}/`;
+    }
 }
 
+// Set default values to contributor friendly values to make development work out of the box without an .env file
+const expensifyCashURL = wrapWithBackslash(lodashGet(Config, 'URL_EXPENSIFY_CASH', 'https://expensify.cash/'));
+const expensifyURL = wrapWithBackslash(lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/'));
+const ngrokURL = wrapWithBackslash(lodashGet(Config, 'NGROK_URL', ''));
+const useNgrok = lodashGet(Config, 'USE_NGROK', 'false') === true;
+const useWebProxy = lodashGet(Config, 'USE_WEB_PROXY', 'true') === true;
```

Comment:
These gotta be checks for a string of `'true'`

Reply:
🤦